cmake/modules/FindSmctemp.cmake

```diff
@@ -0,0 +1,45 @@
+#.rst:
+# FindSmctemp
+# -------
+# Finds the smctemp library
+#
+# This will define the following variables::
+#
+# SMCTEMP_FOUND - system has smctemp
+# SMCTEMP_INCLUDE_DIRS - the smctemp include directory
+# SMCTEMP_LIBRARIES - the smctemp libraries
+# SMCTEMP_DEFINITIONS - the smctemp definitions
+#
+# and the following imported targets::
+#
+#   SMCTEMP::SMCTEMP   - The smctemp library
```

Comment:
The only thing we want to make available is the target, drop all the variables from the comment